[2023-11-05T16:25:26.922+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_People_Places.load_people manual__2023-11-05T16:25:20.775244+00:00 [queued]>
[2023-11-05T16:25:26.945+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_People_Places.load_people manual__2023-11-05T16:25:20.775244+00:00 [queued]>
[2023-11-05T16:25:26.947+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-11-05T16:25:26.950+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-11-05T16:25:26.951+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-11-05T16:25:26.980+0000] {taskinstance.py:1309} INFO - Executing <Task(DockerOperator): load_people> on 2023-11-05 16:25:20.775244+00:00
[2023-11-05T16:25:26.990+0000] {standard_task_runner.py:55} INFO - Started process 663 to run task
[2023-11-05T16:25:26.995+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ETL_People_Places', 'load_people', 'manual__2023-11-05T16:25:20.775244+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/ETL_people_places_dag.py', '--cfg-path', '/tmp/tmpr8de6sd5']
[2023-11-05T16:25:27.001+0000] {standard_task_runner.py:83} INFO - Job 18: Subtask load_people
[2023-11-05T16:25:27.128+0000] {task_command.py:389} INFO - Running <TaskInstance: ETL_People_Places.load_people manual__2023-11-05T16:25:20.775244+00:00 [running]> on host 526ca62ab683
[2023-11-05T16:25:27.322+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_People_Places
AIRFLOW_CTX_TASK_ID=load_people
AIRFLOW_CTX_EXECUTION_DATE=2023-11-05T16:25:20.775244+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-05T16:25:20.775244+00:00
[2023-11-05T16:25:27.369+0000] {docker.py:306} INFO - Starting docker container from image load_people:latest
[2023-11-05T16:25:28.143+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=ETL_People_Places, task_id=load_people, execution_date=20231105T162520, start_date=20231105T162526, end_date=20231105T162528
[2023-11-05T16:25:28.238+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-11-05T16:25:28.281+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
